src/etc/telegraf/custom_scripts/show_interfaces_input_filter.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+import subprocess
+import time
+
+def status_to_int(status):
+    switcher={
+        'u':'0',
+        'D':'1',
+        'A':'2'
+        }
+    return switcher.get(status,"")
+
+def description_check(line):
+    desc=" ".join(line[3:])
+    if desc == "":
+        return "empty"
+    else:
+        return desc
+
+def gen_ip_list(index,interfaces):
+    line=interfaces[index].split()
+    ip_list=line[1]
+    if index < len(interfaces):
+        index += 1
+        while len(interfaces[index].split())==1:
+            ip = interfaces[index].split()
+            ip_list = ip_list + " " + ip[0]
+            index += 1
+            if index == len(interfaces):
+                break
+    return ip_list
+
+interfaces = subprocess.check_output("/usr/libexec/vyos/op_mode/show_interfaces.py --action=show-brief", shell=True).decode('utf-8').splitlines()
```

Comment:
For this we do have an environment variable, `vyos_op_scripts_dir`. Or telegraf runs scripts in a way that cannot read the normal shell environment?

Reply:
Also, does it send that output unchanged? That script has a header. Shouldn't we make a machine-readable output option for it? I'm happy to assist.

```
Codes: S - State, L - Link, u - Up, D - Down, A - Admin Down
Interface        IP Address                        S/L  Description
---------        ----------                        ---  -----------
```